var/spack/repos/builtin/packages/neovim/package.py

```diff
@@ -18,6 +18,9 @@ class Neovim(CMakePackage):
 
     version("master", branch="master")
     version("stable", tag="stable", commit="d772f697a281ce9c58bf933997b87c7f27428a60")
```

Comment:
Meanwhile, 0.11.1 was released. And probably the commit for stable should be removed or updated.
```suggestion
    version("stable", tag="stable")
    version("0.11.1", sha256="ffe7f9a7633ed895ff6adb1039af7516cd6453715c8889ad844b6fa39c3df443")
```

Reply:
Thanks @fthaler! You're right.

I would drop the commit for `stable` branch as you suggested. About `0.11.1` have you had the chance to check if there is any dependency related change?

Reply:
Yes, I checked the dependencies, they are the same as for `0.11.0`.